README.md

```diff
@@ -16,3 +16,81 @@ https://github.com/dwyl/intellectual-property and raise an
+2. Think of the keywords associated with the story/issue you want to create.
+3. Perform a quick search through the existing issues for the project with the keywords you identified e.g: https://github.com/dwyl/time/issues?utf8=%E2%9C%93&q=is%3Aissue%20email if email was your keyword.
+4. Once you are (quite) sure there is no existing issue/story covering what you want to solve/create move on to creating ... (see: [steps below](#issues))
+
+### Labels
+
+GitHub can be a _great_ project management tool if used _effectively_. One of the ways you can manage your project backlog is through **labels**. Labels are a great way to **organise** and **track** the progress of a given project at _any_ point in time.
+
+At dwyl we have created our _own_ standard list of labels that we use for each of our projects. This gives our developers the ability to move between projects **quickly** and **productively**, saving time and energy in the process.
+
+Here is our **[complete list](https://github.com/dwyl/contributing/labels)** of labels and how they should be used:
+
+![labels](https://cloud.githubusercontent.com/assets/12450298/18248682/afcd6974-7371-11e6-84bf-0cb9f4677d92.png)
+
+- `bug` #EE0701 - report a bug that needs fixed
+- `duplicate` #CCCCCC - duplicate issue
+- `enhancement` #84B6EB - improving existing code
+- `help-wanted` #128A0C - looking for help or expertise on a subject
+- `in-progress` #009688 - to be added when an issue has been assigned
+- `in-review` #128A0C - once a PR has been submitted relating to the issue
+- `invalid` #E6E6E6 - GitHub standard tag (not used)
+- `please-test` #08E700 - added after PR is merged (assign to product owner)
+- `priority-1` #0D47A1 - drop everything and work on this (used only when _completely_ neccessary)
+- `priority-2` #1976D2 - high priority issue (what needs doing now)
+- `priority-3` #42A5F5 - high priority (what needs doing next)
+- `priority-4` #8DC9F9 - low priority (to be upgraded later)
+- `priority-5` #C5DEF5 - lowest priority (non-urgent changes)
+- `question` #CC317C - for open questions and discussions
+- `T[x]d` #F06292 - time in 'x' days it should take (estimation)
+- `T[x]h` #F7C6C7 - time in 'x' hours it should take (estimation)
+- `technical` #D4C5F9 - technical issue for developers
+- `wontfix` #ffffff - when an issue won't be addressed (with reason why in comment)
+
+### Issues
+
+At dwyl we use _issues_ to log our user stories and technical tasks for a given project. User stories are higher level issues that describe certain functionalities of the application you're building. These user stories get broken down into smaller **_technical_** tasks. Here is our work flow step-by-step for creating and developing a user story and its technical tasks:
+
+**NOTE:** Please avoid working on something before a relevant issue has been opened. Once the issue has been discussed and accepted then work can begin to solve it.
```

Comment:
There _must_ be a comment from at least one other person on the issue _acknowledging_ it and/or _confirming_ the requirement/bug before work starts. This is necessary to avoid doing _unnecessary_ work. e.g. someone may have already solved that problem or implemented a similar feature on a different project and can point you in the direction of sample code or the PO/Client may deem the issue/story to be _lower priority_ than something else on the list.